### DISCUSSION TITLE
Documentation for v1.3.6

### DISCUSSION THREAD

**Discussion by leanjunio** [js-fns/date-fns#2465] [Q&A]
Hey guys,

Are there any documentations available for v1.3.6? I couldn't find it on the site. Though I do so v.1.30.1 twice.

**Comment by tan75** [comment]
> Hey guys,
> 
> Are there any documentations available for v1.3.6? I couldn't find it on the site. Though I do so v.1.30.1 twice.

hi @leanjunio 
Yes you are right! I can not see the documentation for the version 1.3.6. either however, I will try to investigate this.
Just wondering, is there a reason for using such an old version of the date-fns? Where did you download it from? I couldn't find it on npm

**Comment by fturmel** [comment]
Very late answer, but I opened an issue on the site for the dupe v1.30.1 problem: https://github.com/date-fns/date-fns.org/issues/207

As far as v1.3.6 goes, there are no releases with that semver on npm or github's tags. Maybe it's a typo?
https://www.npmjs.com/package/date-fns?activeTab=versions
https://github.com/date-fns/date-fns/tags

<img width="526" alt="Screen Shot 2021-12-07 at 9 50 03 AM" src="https://user-images.githubusercontent.com/4009209/145051202-0e6fc07f-14c4-4ea5-a64a-e6441b27c496.png">
